subsys/bluetooth/mesh/cfg_cli.c

```diff
@@ -1826,7 +1826,10 @@ static int mod_sub(uint32_t op, uint16_t net_idx, uint16_t addr, uint16_t elem_a
 
 	bt_mesh_model_msg_init(&msg, op);
 	net_buf_simple_add_le16(&msg, elem_addr);
-	net_buf_simple_add_le16(&msg, sub_addr);
+
+	if (sub_addr != BT_MESH_ADDR_UNASSIGNED) {
```

Comment:
I found in spec:

>4.3.2.26 Config Model Subscription Status
>...
>The value of the Address field shall contain the address that was used to modify the Subscription List or the unassigned address. When referencing the Label UUID, the virtual address shall be used. The value of the Address field shall not be a unicast address or the all-nodes address.
>...

Why did you omit the field? Looks like the unassigned address is relevant type.

Reply:
`mod_sub` is used to generate messages `Subscription (Virtual Address) Add`, `Subscription (Virtual Address) Delete`, `Subscription (Virtual Address) Overwrite` and `Subscription Delete All`. Only `Subscription Delete All` message doesn't have address field in the body. Status message is processed in `mod_sub_status` function.